docs/configuration.md

```diff
@@ -1398,6 +1398,15 @@ Apart from these, the following properties are also available, and may be useful
   </td>
 </tr>
 <tr>
+  <td><code>spark.scheduler.listenerbus.eventqueue.size</code></td>
+  <td>10000</td>
+  <td>
+    Capacity for event queue in Spark listener bus, must be greater than 0. Consider increasing
+    value (e.g. 20000) if listener events are dropped, because of the rate at which tasks are being
```

Comment:
thanks for updating, sorry a couple minor nits.  

I don't think we need the line "because of the rate at which tasks are being started by the scheduler;"  

Perhaps clarify the last bit about memory: " Increasing this value will results in the driver using more memory."